assets/dev/scss/admin/_experiments.scss

```diff
@@ -5,6 +5,7 @@
 		display: flex;
 		align-items: flex-start;
 		flex-wrap: wrap;
```

Comment:
Is flex-wrap necessary when using direction column? If not, can you please remove it?